runsc/boot/compat_test.go

```diff
@@ -32,54 +30,3 @@ func TestOnceTracker(t *testing.T) {
 		}
 	}
 }
-
-func TestArgsTracker(t *testing.T) {
```

Comment:
You could keep the bulk of the test in this file. Replace rdi/rsi with arg1/arg2 and create a function to convert args into registers in the arch specific file.

Reply:
Good idea! I will replace them in the next patch.